docs/lib/snippets/modular/drift/example.drift.dart

```diff
@@ -42,6 +42,7 @@ class Todos extends i0.Table with i0.TableInfo<Todos, i1.Todo> {
   String get aliasedName => _alias ?? 'todos';
   @override
   String get actualTableName => 'todos';
+  static String get actualTableNameStatic => 'todos';
   @override
```

Comment:
I'm not sure if `actualTableNameStatic` is such a good name for this, but I also can't come up with something much better. Perhaps we can call it `$name`?

I think it would also be nice to have this as a constant field instead of a getter. To avoid repetition, I think we should also modify `actualTableName` and `aliasedName` to return `actualTableName` (or `$name`) instead of duplicating the string literal.

Reply:
Yeah... I was also not happy with `actualTableNameStatic` 😅  ... 

I renamed it in `$name` in [revamp: Rename actualTableNameStatic into $name](https://github.com/simolus3/drift/pull/2625/commits/d001ad533f91b990e13f3fd10a4c948a26b4f221)